internal/resources/providers/azurelib/inventory/storage_provider.go

```diff
@@ -152,6 +180,62 @@ func (p *storageAccountProvider) ListStorageAccountBlobServices(ctx context.Cont
 	return assets, nil
 }
 
+func (p *storageAccountProvider) ListStorageAccountQueues(ctx context.Context, storageAccounts []AzureAsset) ([]AzureAsset, error) {
+	var assets []AzureAsset
+	for _, sa := range storageAccounts {
+		responses, err := p.client.AssetQueues(ctx, sa.SubscriptionId, nil, sa.ResourceGroup, sa.Name, nil)
+		if err != nil {
+			return nil, fmt.Errorf("error while fetching azure queues for storage accounts %s: %w", sa.Id, err)
+		}
+
+		queues, err := transformQueues(responses, sa)
+		if err != nil {
```

Comment:
Same comment as StorageQueueService, should we stop the execution if the transform queue can't extract properties?

Reply:
Makes sense 👍